conf/pacific/rbd/tier-0_rbd.yaml

```diff
@@ -28,6 +28,7 @@ globals:
         role:
           - mds
           - osd
+          - rgw
```

Comment:
Please cover this daemon and test in quincy as well.

Reply:
Done.